app/database/managers/schema_manager.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import SQLiteAdapter from '@nozbe/watermelondb/adapters/sqlite';
+import {schemaMigrations} from '@nozbe/watermelondb/Schema/migrations';
+import {defaultSchema} from '../default/schema';
+
+/**
+ * Creates an adapter options object that will be used by the database manager.
+ * @param {string} dbPath - The dbName or filePath (e.g. AppGroup path) for the database
```

Comment:
I rather separate this two as in `path` and `name` and then construct the `dbName` based on those two values where  `path` default value is an empty string and `name` is *default*